assets/src/components/notifications.tsx

```diff
@@ -30,7 +30,10 @@ export const Notifications = () => {
 
   useEffect(() => {
     if (notificationToShow && window.FS) {
-      if (notificationToShow.reason in chelseaReasons) {
+      if (
+        notificationToShow.reason === "chelsea_st_bridge_lowered" ||
+        notificationToShow.reason === "chelsea_st_bridge_raised"
+      ) {
```

Comment:
What's the reason for avoiding the use of `chelseaReasons`?

Reply:
As I've tripped over before, `in` uses `==`, which means that it is basically useless with strings--you need `===` to compare equality with strings in the sane way.

Reply:
Even if you avoid `in`, can't you do something like `chelseaReasons.some((reason) => reason === notificationToShow.reason)` rather then re-writing the string values?

Reply:
hmmm yes that would work